server.py

```diff
@@ -27,12 +31,30 @@ def parse_arguments():
 if __name__ == '__main__':
     parse_arguments()
 
-    print(f"Config Directory: {instance_utils.CONFIG_DIRECTORY}")
-    print(f"User: {instance_utils.USER}")
-
-    print(f"Deleting old locks that do not belong to Wilmer Instance_Id: '{instance_utils.INSTANCE_ID}'")
+    logging.basicConfig(
+        handlers=[
+            logging.StreamHandler(),
+            RotatingFileHandler(
+                "logs/wilmerai.log",
+                maxBytes=1048576 * 5,
+                backupCount=7,
+            ),
+        ],
+        level=logging.DEBUG,
+        format="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+    )
```

Comment:
I'd like to update this section to add the handlers based on a user's preference in their user file. 

Could you take a peek at these recommendations below?

A) add "useFileLogging" to the user file, as a boolean

B) Update the code above to something like this, which will also ensure that the log folder exists, and also reduced the file log size down to 3mb because opening a 5mb text file is horrifying on Windows lol:

```
    use_file_logging = config_utils.get_use_file_logging()
    handlers = []

    # Add handlers based on user file -> useFileLogging
    if not use_file_logging:
        handlers.append(logging.StreamHandler())
    else:
        if not os.path.exists("logs"):
            os.makedirs("logs")

        handlers.append(logging.StreamHandler())
        handlers.append(RotatingFileHandler(
            "logs/wilmerai.log",
            maxBytes=1048576 * 3,
            backupCount=7,
        ))

    # Configure logging with the selected handlers
    logging.basicConfig(
        handlers=handlers,
        level=logging.DEBUG,
        format="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s",
        datefmt="%Y-%m-%d %H:%M:%S",
    )
```


C) Add the following to config_utils somewhere:

```
def get_use_file_logging():
    """
    Retrieves the user's preference of whether to log wilmer logs to a file or just the console.

    :return: Boolean signifying whether to log to a file or not.
    """
    return get_config_value('useFileLogging')
```

Reply:
Done